index.d.ts

```diff
@@ -0,0 +1,34 @@
+// Definitions by: Jannik <https://github.com/jannikkeye>
+//                 Leo <https://github.com/leomelzer>
```

Comment:
This was released under a given license. We should maintain that license, maybe even in the LICENSE file.

Reply:
DefinitelyTyped typings are released under [MIT](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/LICENSE). I am not sure I understand what you mean by maintaining the license then. I am not sure how to proceed here as the typings have been modified to use fastify.Plugin for the plugin definition. See [original typings](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/fastify-static/index.d.ts) and [modified typings](https://github.com/fastify/fastify-static/blob/4ab038ed54e8776074004f11de883752677d541a/index.d.ts). Basically only all but one option remain unchanged.

Reply:
This is ok, It’s me being paranoid about licensing and including the correct one.